src/main/resources/init.sql

```diff
@@ -0,0 +1,24 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE USERS
+(
+    ID UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+    USERNAME VARCHAR(200) UNIQUE NOT NULL,
+    NAME VARCHAR(200) NOT NULL,
+    SURNAME VARCHAR(200) NOT NULL,
+    EMAIL VARCHAR(100) NOT NULL,
+    PHONE_NUMBER VARCHAR(15),
+    ADDRESS VARCHAR(300),
+    UPDATED_AT TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
```

Comment:
why do you have two different files with the same logic?

Reply:
+